shared-bindings/vectorio/__init__.c

```diff
@@ -0,0 +1,46 @@
+#include <stdint.h>
+
+#include "py/obj.h"
+#include "py/runtime.h"
+
+#include "shared-bindings/vectorio/Circle.h"
+#include "shared-bindings/vectorio/Polygon.h"
+#include "shared-bindings/vectorio/Rectangle.h"
+#include "shared-bindings/vectorio/VectorShape.h"
+
+//| :mod:`vectorio` --- Lightweight 2d shapes for displays
+//| =========================================================================
+//|
+//| .. module:: vectorio
+//|   :synopsis: Adds vector graphics to displayio
+//|   :platform: SAMD21, SAMD51, nRF52
```

Comment:
Any reason this can't work on imxrt?

Reply:
Currently I'm defaulting this to build where `displayio` builds.  `displayio` claims these platforms in its docs.  If you have `displayio` you _should_ have `vectorio` unless the pending build shows we need to disable some platforms (e.g., due to flash space)

Reply:
The docs are likely just wrong.